roles/idrac_storage_controller/molecule/__fetch_volumes.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Pre_req - View all volumes
+  dellemc.openmanage.idrac_storage_volume:
+    idrac_ip: "{{ lookup('env', 'IDRAC_IP') }}"
+    idrac_user: "{{ lookup('env', 'IDRAC_USER') }}"
+    idrac_password: "{{ lookup('env', 'IDRAC_PASSWORD') }}"
+    validate_certs: false
+    state: "view"
+  register: result
```

Comment:
Can we register the var on the format role_name_var, PLease update other files as required.

Reply:
Updated